.github/workflows/controller-test.yml

```diff
@@ -0,0 +1,31 @@
+name: Controller Integration Tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+    - 'release/*'
+
+env:
+  MAIN_BRANCH: origin/master
```

Comment:
what's the `MAIN_BRANCH` env var doing? Do we really need it?

Reply:
Removed